firstissue.json

```diff
@@ -1,1310 +1,1325 @@
-    "uber/kraken",
-    "ubuntu/microk8s",
-    "unisonweb/unison",
-    "UniversalMediaServer/UniversalMediaServer",
-    "unoplatform/uno",
-    "uutils/coreutils",
-    "valhalla/valhalla",
-    "validatorjs/validator.js",
-    "Varying-Vagrant-Vagrants/VVV",
-    "vectordotdev/vector",
-    "vercel/async-retry",
-    "vercel/commerce",
-    "vercel/examples",
-    "vercel/hazel",
-    "vercel/hyper",
-    "vercel/ncc",
-    "vercel/next-learn",
-    "vercel/next.js",
-    "vercel/nextjs-subscription-payments",
-    "vercel/og-image",
-    "vercel/pkg",
-    "vercel/platforms",
-    "vercel/satori",
-    "vercel/serve",
-    "vercel/styled-jsx",
-    "vercel/swr",
-    "vercel/turbo",
-    "vercel/vercel",
-    "vercel/virtual-event-starter-kit",
-    "vesoft-inc/nebula",
-    "vespa-engine/vespa",
-    "vicky002/AlgoWiki",
-    "VictoriaMetrics/VictoriaMetrics",
-    "vimeo/psalm",
-    "virtual-kubelet/virtual-kubelet",
-    "vitessio/vitess",
-    "vmware-tanzu/carvel-ytt",
-    "vmware-tanzu/community-edition",
-    "vmware-tanzu/octant",
-    "vmware-tanzu/sonobuoy",
-    "vmware-tanzu/velero",
-    "vmware/differential-datalog",
-    "vmware/govmomi",
-    "vmware/open-vm-tools",
-    "vmware/photon",
-    "vmware/pyvmomi",
-    "volcano-sh/volcano",
-    "volta-cli/volta",
-    "VSCodeVim/Vim",
-    "vuejs/vue",
-    "vuejs/vue-router",
-    "vuejs/vuepress",
-    "vuetifyjs/vuetify",
-    "VulcanJS/Vulcan",
-    "wagtail/wagtail",
-    "wasm3/wasm3",
-    "WasmEdge/WasmEdge",
-    "wasmerio/wasmer",
-    "wasp-lang/wasp",
-    "weaveworks/eksctl",
-    "weaveworks/ignite",
-    "weaveworks/weave",
-    "web-platform-tests/wpt",
-    "WebAssembly/spec",
-    "WebAssembly/WASI",
-    "webhintio/hint",
-    "webiny/webiny-js",
-    "WeblateOrg/weblate",
-    "wemake-services/wemake-python-styleguide",
-    "werf/werf",
-    "wesnoth/wesnoth",
-    "wger-project/wger",
-    "whatwg/fetch",
-    "whatwg/html",
-    "whitesmith/rubycritic",
-    "winstonjs/winston",
-    "winsw/winsw",
-    "woocommerce/woocommerce",
-    "woodpecker-ci/woodpecker",
-    "wordpress-mobile/WordPress-Android",
-    "wordpress-mobile/WordPress-iOS",
-    "WordPress/gutenberg",
-    "wp-graphql/wp-graphql",
-    "xenia-project/xenia",
-    "xtdb/xtdb",
-    "xtermjs/xterm.js",
-    "xunit/xunit",
-    "yarnpkg/yarn",
-    "Yelp/paasta",
-    "yewstack/yew",
-    "Yoast/wordpress-seo",
-    "yugabyte/yugabyte-db",
-    "zabbix/zabbix",
-    "zalando/skipper",
-    "zaproxy/zaproxy",
-    "zcash/zcash",
-    "zeebe-io/zeebe",
-    "zeek/zeek",
-    "zeit/hyper",
-    "zeit/next.js",
-    "zephyrproject-rtos/zephyr",
-    "zeromicro/go-zero",
-    "zio/zio",
-    "zitadel/zitadel",
-    "zsh-users/zsh-completions",
-    "zsh-users/zsh-syntax-highlighting",
-    "zulip/zulip"
-  ],
-  "labels": [
-    "good first issue",
-    "help wanted"
-  ]
-}
+[
+  {
+    "name": "GitLab",
+    "provider": "gitlab",
+    "repositories": [
+      "gid://gitlab/Project/278964"
```

Comment:
**todo:** Either here or in a follow-up, it'd be nice to improve the selector we use for GitLab repositories.

As a developer wanting to add a GitLab repository to this project, it's hard for me to tell what projects are already included since we are looking by ID. It looks like the GraphQL API doesn't support querying by multiple project paths? In the meantime maybe we could do something like `"gitlab-org/gitlab|278964"` and split on `|` to grab the ID. This way we are still capturing the project path in the meantime.

We don't have to handle this here. Just dropping a note for this.

Reply:
Thanks for the note Paul 🙂 That's actually one thing I have discussed with Lee today and we decided to use the ID until we add the possibility to filter by multiple project paths on the GitLab side 🙂 

I like your splitting idea though 🤔 I'll give it a shot (it's just a string split 😄 )

Reply:
The ID is fine for now, but make sure to create a follow-up issue to replace that with the repo name

Reply:
@lucavallin I applied Paul's suggestion so that once we improve the GitLab endpoint we just have to remove the ID and leave the project full path (so we don't have to manually find it from the ID) 👍 🙂

Reply:
Follow-up issue created 👍